spring-cloud-streams/src/main/java/org/springframework/cloud/streams/config/ChannelBindingAdapterConfiguration.java

```diff
@@ -49,7 +49,7 @@
  *
  */
 @Configuration
-@ImportResource("classpath*:/META-INF/spring-xd/bus/codec.xml")
+@ImportResource("classpath*:/META-INF/spring-bus/codec.xml")
```

Comment:
I think the XML config goes away. It's been removed from XD already.  The Rabbit and Redis MB configurations are XML that have <constructor-arg ref="codec"/>  I would like to get rid of the magic bean name and Autowire by type instead. Then it should be a matter of adding a jar that provides an implementation of MultitypeCodec.